.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: Elixir CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Build and test
+    environment: dwylauth
```

Comment:
Removing this line as it's a hangover from copying this file from the `auth` project. 👌 
```suggestion
```